configuration/configuration.go

```diff
@@ -266,10 +266,12 @@ func assertConstructionConfiguration(ctx context.Context, config *ConstructionCo
 			return fmt.Errorf("Account.Address is missing for prefunded account")
 		}
 
-		// Check if valid Currency
-		err = asserter.Currency(account.Currency)
-		if err != nil {
-			return fmt.Errorf("%w: invalid currency for prefunded account", err)
+		// Check if valid Currency when Currency is specified
```

Comment:
Why we do this

Reply:
This is for the case where `config.PrefundedAccounts.account.Currency == nil`, in that case we shouldn't validate the currency, and the successive `/account/balance` API will return the balances of all accounts for the specific prefunded account

Reply:
I suggest you add comments on what happens if currency is not specified

Reply:
Sure, will add it